include/SFML/System/Time.inl

```diff
@@ -213,34 +213,39 @@ constexpr Time& operator*=(Time& left, std::int64_t right)
 ////////////////////////////////////////////////////////////
 constexpr Time operator/(Time left, float right)
 {
+    assert(right != 0 && "Cannot divide by zero");
     return seconds(left.asSeconds() / right);
 }
 
 
 ////////////////////////////////////////////////////////////
 constexpr Time operator/(Time left, std::int64_t right)
 {
+    assert(right != 0 && "Cannot divide by zero");
     return microseconds(left.asMicroseconds() / right);
 }
 
 
 ////////////////////////////////////////////////////////////
 constexpr Time& operator/=(Time& left, float right)
 {
+    assert(right != 0 && "Cannot divide by zero");
     return left = left / right;
 }
 
 
 ////////////////////////////////////////////////////////////
 constexpr Time& operator/=(Time& left, std::int64_t right)
 {
+    assert(right != 0 && "Cannot divide by zero");
     return left = left / right;
 }
 
 
 ////////////////////////////////////////////////////////////
 constexpr float operator/(Time left, Time right)
 {
+    assert(right.asMicroseconds() != 0 && "Cannot divide by zero");
```

Comment:
Should these also mentioned the function name?

Reply:
We do that in Vector2 and Vector3 but not elsewhere. I made sure to keep the vectors internally consistent but we have no established convention for whether assertions in public APIs ought to name the public API they're contained within.

Reply:
Would you like to establish a convention about whether or not we do this? I'm open to that idea.